engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/DebugLocalScope.java

```diff
@@ -188,15 +205,78 @@ boolean isMemberReadable(String memberName) {
   @ExportMessage
   @TruffleBoundary
   Object readMember(String member, @CachedLibrary("this") InteropLibrary interop)
-      throws UnknownIdentifierException {
+      throws UnknownIdentifierException, UnsupportedMessageException {
     if (!allBindings.containsKey(member)) {
       throw UnknownIdentifierException.create(member);
     }
     FramePointer framePtr = allBindings.get(member);
     var value = getValue(frame, framePtr);
+    if (value != null && isHostObject(value)) {
+      // Determine if the host object can be converted to corresponding Enso value.
+      var interopUncached = InteropLibrary.getUncached();
```

Comment:
What is we just invoke `value.to Any` or some other well known method invocation on the value? Unless I am mistaken most of the conversions happen for `self` argument before the dispatch happens.

Reply:
Implemented via your suggestion in 19e37984ff41e9ef7cd982d2e5d46ab374a6f252